cmd/sql/flow.go

```diff
@@ -125,6 +126,15 @@ func buildFlagsAndMountDirs(projectDir string, setProjectDir, setAirflowHome, se
 		mountDirs = append(mountDirs, airflowDagsFolderAbs)
 	}
 
+	if setDataDir && dataDir != "" {
+		dataDirAbs, err := getAbsolutePath(dataDir)
+		if err != nil {
+			return nil, nil, err
+		}
+		flags["data-dir"] = dataDirAbs
+		mountDirs = append(mountDirs, dataDirAbs)
```

Comment:
It would be great if we could add a test covering this new behaviour - but perhaps we can cover this in the E2E test, WDYT @feluelle ?

Reply:
This is being covered. The line it complains about is in case `getAbsolutePath` raises an error.